demisto_sdk/commands/validate/tests/BA_validators_test.py

```diff
@@ -2418,3 +2421,73 @@ def test_IsTestsSectionValidValidator_obtain_invalid_content_items():
             for result, expected_msg in zip(results, expected_msgs)
         ]
     )
+
+
+def test_is_command_or_script_name_starts_with_digit_invalid():
+    """
+    Given
+    - One invalid integration with one command starting with a digit character.
+    When
+    - Calling the IsCommandOrScriptNameStartsWithDigitValidator obtain_invalid_content_items function.
+    Then
+    - Make sure one failure is returned and the error message is correct.
+    """
+    with ChangeCWD(REPO.path):
```

Comment:
not sure you need the ChangeCWD part.

Reply:
You're right! Removed from my unit tests in commit ceaef8a04f169a5a95f9188fd1f16c66ef316110 (and needs to be removed in many others as well)